test/async-hooks/test-graph.tcp.js

```diff
@@ -17,7 +17,7 @@ const server = net
 
 server.listen(common.PORT);
 
-net.connect({ port: server.address().port, host: server.address().address },
+net.connect({ port: server.address().port, host: '::1' },
```

Comment:
I don't think `localhost` will work because of https://github.com/nodejs/node/issues/6307